books/models.py

```diff
@@ -237,7 +239,7 @@ def clean(self):
 
 class StudentResources(models.Model):
     resource = models.ForeignKey(
-        StudentResource,
+        snippets.StudentResource,
```

Comment:
If you wrap this in quotes, you don't need to import the models

```
resource = models.ForeignKey(
        'snippets.StudentResource',
```

Reply:
I like the constants file idea - good solution.. but it could still live here since you won't have to import snippet models if you do this

Reply:
documentation: https://docs.djangoproject.com/en/3.2/ref/models/fields/#module-django.db.models.fields.related

scroll down a little bit to where it talks about circular dependencies

Reply:
I have to import snippets because only ForeignKey works with the package.Model string. There is a SharedContent block and the new errata_content function that need the import to work.